css.html.markdown

```diff
@@ -13,12 +13,11 @@ further development of browsers, fully visual web pages also became common.
 CSS is the standard language that exists to keep the separation between   
 the content (HTML) and the look-and-feel of web pages.
 
+The latest version of CSS is CSS3. Before using the CSS3 properties, check if your browser is compatible with CSS3.
```

Comment:
There is a link at the bottom to check the compatibility of your browser (caniuse.com). Hence I did not mention the same again.